src/variables.jl

```diff
@@ -176,6 +176,22 @@ function delete(model::Model, variable_ref::VariableRef)
     MOI.delete(backend(model), variable_ref.index)
 end
 
+"""
+    delete(model::Model, variable_refs::VariableRef)
```

Comment:
The signature doesn't match. Should be `Vector{VariableRef}`.

Reply:
Fixed.